src/plots/cartesian/axes.js

```diff
@@ -250,16 +250,16 @@ var getDataConversions = axes.getDataConversions = function(gd, trace, target, t
 
     // if 'target' has corresponding axis
     // -> use setConvert method
-    if(ax) return {d2c: ax.d2c, c2d: ax.c2d};
+    if(ax) return { d2c: ax.d2c, c2d: ax.c2d };
```

Comment:
Would you mind reverting all the linting changes in this commit?

Reply:
Done